snippets/16-rubric.md

```diff
@@ -1,21 +1,22 @@
 There are two levels of access to OpenSAFELY available for researchers:
 
-* **access to the OpenSAFELY results server**
-    * for review of OpenSAFELY study results before release
+* **access to the OpenSAFELY results server (also referred to as the Level 4 environment)**
+    * where you review your study results before applying disclosure controls and requesting their release
 * **access to OpenSAFELY's GitHub organisation**
-    * for submission of studies to OpenSAFELY
+    * where you write your study code, which is then executed against the patient data held on the results server; where you view results that have been released from the results server (after disclosure controls have been applied)
```

Comment:
Nitpick: Possibly *where you store your study code*: a researcher can _write_ their study code on their computer; or on someone else's computer ("the cloud"). However, to work with OpenSAFELY, this code has to be *stored* within the GitHub organization.